www-servers/lighttpd/files/lighttpd.initd

```diff
@@ -4,7 +4,7 @@
 
 extra_started_commands="reload graceful"
 
-LIGHTTPD_PID="$(grep pid ${LIGHTTPD_CONF} | cut -d '=' -f 2 | tr -d \\\" | tr -d [:space:])"
```

Comment:
Still modifying the stable copy? Update the/a new one, I think.

Reply:
I must admit that I haven't set up a whole VM for every build system to which I try to contribute. What are the underlying commands that the ebuild command magically does for me?

Reply:
... although if you hadn't had a chance to try this on a real system (doing it blind or something?) do say, as it means I should test a fair bit more.

Reply:
Yes, I have not tested on a gentoo system.  However, what I am doing here is reverting to the same value used previously in gentoo -- which gentoo should have reverted back to use at the end of 2016 -- and is valid for all lighttpd-*.ebuild versions currently in the repo.

Reply:
No worries.

Reply:
A note to close this out: unless you request otherwise, I am planning to leave these changes in lighttpd.initd, as it applies to all current ebuild versions.

Reply:
Changed just the new version so stable versions aren't affected.